src/components/headerComponent.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+    <header id="header">
```

Comment:
Don't use id header for a tag that is already semantically header. Just use header or give it another name that is more specific to your usage of it.